fictionary/__init__.py

```diff
@@ -15,29 +15,36 @@
 import random
 import shelve
 import sys
+import tempfile
 
-import click
-
-APP_NAME = "fictionary"
 
 # Where to save the generated data file:
-DATA_FILE_ROOT = click.get_app_dir(APP_NAME)
+DATA_FILE_ROOT = tempfile.gettempdir()
```

Comment:
This change is actually incorrect. The data files should be persisted between runs of this application. tempdirs hold no such guarantee. Please can you roll back this change.

Reply:
I made this change mainly because I feel that having click as a dependency is overkill, just so that one little click function `get_app_dir()` can be used. I've now changed it so that it conditionally tries to import click and to use the value of `click.get_app_dir(APP_NAME)`, and when that fails, it falls back to `tempfile.gettempdir()`. And I have documented click as an optional dependency in the README. Hope you're happy with that?